Add the list of the supported column types in the editing. (#10655)
camptocamp/c2cgeoportal: doc/administrator/editing.rst

       }
   }

7. The supported column types are:

      * ``integer``
      * ``numeric``
      * ``character varying``
      * ``text``
      * ``boolean``
      * ``date``
      * ``time without time zone``
      * ``timestamp without time zone``

>    Note that th following types will have an issue with the timezone:

Reviewer: ```suggestion
   Note that the following types will have an issue with the timezone:
```

>    Note that the following types will not working correctly with the timezone:

Reviewer 2: The sentence is grammatically incorrect,
but more importantly: it is not clear why a type "with time zone" will not work correctly with the timezone. Is it a bug in GMF? Or what exactly is not working?

Author: GeoMapFish first implementation uses the types `... without time zone` then the other one needs a specific development....

Reviewer 2: So are we saying the following?
"Note that the following timezone related types are currently not supported:"

Author: I don't understand your sentences...
We provide a list of types not of timezones...

Reviewer: Ok so:

```suggestion"
   Note that the following timezone aware types are currently not supported:
```
